lib/templates/typedef.html

```diff
@@ -8,7 +8,7 @@ <h5>{{parent.name}} {{parent.kind}}</h5>
 
   <div id="dartdoc-main-content" class="col-xs-12 col-sm-9 col-md-8 main-content">
     {{#self}}
-    <div>{{>source_link}}<h1>{{{nameWithGenerics}}} {{kind}} {{>categorization}}</h1></div>
+      <div>{{>source_link}}<h1><span class="kind-typedef">{{{nameWithGenerics}}}</span> {{kind}} {{>categorization}}</h1></div>
```

Comment:
extra space

Reply:
I was making it match the other templates... I'll make them all match: I think they need the indent because the `{{#self}}` wraps it with tags.